modules/nw-multi-network-policy-ipv6-suppport.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-multi-network-policy-ipv6-support_{context}"]
+= Supporting multi-network policies in IPv6 networks
+
+The ICMPv6 Neighbor Discovery Protocol (NDP) plays a crucial role in IPv6 networks, facilitating the interaction between devices on the same link. NDP is a set of messages and processes that enable devices to discover and maintain information about neighboring nodes.
```

Comment:
There's no right or wrong here, but maybe defining what it is before defining its role is worth considering. Definitely up to you how you want to approach it. 

```suggestion
The ICMPv6 Neighbor Discovery Protocol (NDP) is a set of messages and processes that enable devices to discover and maintain information about neighboring nodes. NDP plays a crucial role in IPv6 networks, facilitating the interaction between devices on the same link. NDP 
```